addons/medical_vitals/functions/fnc_handleUnitVitals.sqf

```diff
@@ -165,4 +165,7 @@ if (!isPlayer _unit) then {
 
 END_COUNTER(Vitals);
 
+//placed outside the counter as 3rd-party code may be called from this event
+[QEGVAR(medical,handleUnitVitals), _unit] call CBA_fnc_localEvent;
```

Comment:
Thoughts on providing `_deltaT` here as well for convenience?

Reply:
good call